recipes/server_source.rb

```diff
@@ -30,9 +30,9 @@
 node.default['nagios']['conf']['p1_file'] = nil
 
 pkgs = value_for_platform_family(
-  %w( rhel fedora ) => %w( openssl-devel gd-devel tar ),
-  'debian' => %w( libssl-dev libgd2-xpm-dev bsd-mailx tar ),
-  'default' => %w( libssl-dev libgd2-xpm-dev bsd-mailx tar )
+  %w(rhel fedora) => %w(openssl-devel gd-devel tar),
```

Comment:
IIRC @shoekstra wanted to drop support for Fedora (but I'd suggest we want to keep CentOS)